src/fabric.c

```diff
@@ -437,6 +437,7 @@ void fi_ini(void)
 	ofi_register_provider(VERBS_INIT, NULL);
 	ofi_register_provider(GNI_INIT, NULL);
 	ofi_register_provider(BGQ_INIT, NULL);
+	ofi_register_provider(NETDIR_INIT, NULL);
```

Comment:
Yes, you're right. This build is broken w/o my 2nd PR
But I guess that we're going to merge 2nd PR (#2853) and only after that this one will be merged
Do you agree? or Do I need to move this one into 2nd PR?